-i https://pypi.org/simple
exceptiongroup==1.2.2; python_version < '3.11'
iniconfig==2.0.0; python_version >= '3.7'
packaging==24.2; python_version >= '3.8'
pluggy==1.5.0; python_version >= '3.8'
pytest==8.3.3; python_version >= '3.8'
tomli==2.0.2; python_version < '3.11'
attrs==24.2.0; python_version >= '3.7'
boto3==1.35.57; python_version >= '3.8'
botocore==1.35.57; python_version >= '3.8'
certifi==2025.4.26; python_version >= '3.6'
cfn-resource-provider==1.2.1
charset-normalizer==3.4.2; python_version >= '3.7'
idna==3.10; python_version >= '3.6'
jmespath==1.0.1; python_version >= '3.7'
jsonschema==4.23.0; python_version >= '3.8'
jsonschema-specifications==2024.10.1; python_version >= '3.9'
python-dateutil==2.9.0.post0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2'
referencing==0.35.1; python_version >= '3.8'
requests[security]==2.32.4; python_version >= '3.8'
rpds-py==0.21.0; python_version >= '3.9'
s3transfer==0.10.3; python_version >= '3.8'
six==1.16.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2'
urllib3==1.26.20; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5'
